pkg/chunk/cached_store.go

```diff
@@ -434,8 +435,12 @@ func (s *wSlice) upload(indx int) {
 		if s.store.conf.Writeback {
 			stagingPath, err := s.store.bcache.stage(key, block.Data, s.store.shouldCache(blen))
 			if err != nil {
-				s.store.stageBlockErrors.Add(1)
-				logger.Warnf("write %s to disk: %s, upload it directly", stagingPath, err)
+				logLevel := logrus.DebugLevel
```

Comment:
Don't adjust log level, instead use `Warnf` and `Debugf` in different if blocks.

Reply:
Updated.